packages/terra-alert/tests/jest/Alert.test.jsx

```diff
@@ -5,8 +5,11 @@ import { mountWithIntl } from 'terra-enzyme-intl';
 
 import IconHelp from 'terra-icon/lib/icon/IconHelp';
 import Button from 'terra-button';
+import classNames from 'classnames/bind';
 import Alert from '../../src/Alert';
+import styles from '../../src/terra-dev-site/doc/example/colors.module.scss';
 
+const cx = classNames.bind(styles);
```

Comment:
This doesn't hurt anything, but it is probably overkill to use classnames to hash strings within the jest tests. The un-hashed string should be fine. The hashing can also be customized which might make this test flaky down the road.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2407/commits/016e1a078d9ba50f5cf9c9d7c00f95560540025f.